fastlane_core/lib/fastlane_core/globals.rb

```diff
@@ -0,0 +1,33 @@
+module FastlaneCore
+  class Globals
+    def self.captured_output
+      unless @captured_output
+        @captured_output = ""
+      end
+      @captured_output
+    end
+
+    class << self
+      attr_writer :captured_output
+      attr_writer :capture_output
+      attr_writer :verbose
+    end
+
+    def self.capture_output?
+      return nil unless @capture_output
+      return true
+    end
+
+    def self.captured_output?
+      if @capture_output && @captured_output.length > 0
+        return true
+      end
+      return false
+    end
+
+    def self.verbose?
+      return nil unless @verbose
+      return true
```

Comment:
Why have the extra check here? Is there a reason we want to return nil when the value is false?